03 - Showcases & Templates/Vaults/Template_Hub.md

```diff
@@ -0,0 +1,49 @@
+---
+aliases: Template_Hub
+- 
+tags:
+- seedling
+- Template_Hub
+publish: true
+---
+
+# Template_Hub
+Author: [Hugo Sanots (Zektor)](Zektor)
```

Comment:
Last name

Reply:
-.-! now i just feel stupid -.- sorry about that .... again